secretsanta/cli/cli.py

```diff
@@ -0,0 +1,58 @@
+import click
+import json
+
+from secretsanta.main import funs as secretsanta
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.group(context_settings=CONTEXT_SETTINGS)
+@click.version_option(version='0.0.1')
```

Comment:
Should this not be derived from the `__init__.py` file?

Reply:
yea, this is kinda redundant. let's discuss if we shall just remove it from `__init__.py` or synchronize